packages/core/admin/admin/src/content-manager/pages/ListSettingsView/components/SortDisplayedFields.js

```diff
@@ -36,6 +37,21 @@ const SortDisplayedFields = ({
 }) => {
   const { formatMessage } = useIntl();
   const [isDraggingSibling, setIsDraggingSibling] = useState(false);
+  const [fieldWasAdded, setFieldWasAdded] = useState(false);
+  const fieldRefMap = useRef([]);
+
+  function handleAddField(field) {
+    onAddField(field);
+    setFieldWasAdded(true);
+  }
+
+  useEffect(() => {
+    if (fieldWasAdded) {
+      const lastField = last(fieldRefMap.current);
+
+      lastField.scrollIntoView();
```

Comment:
👀🔥

Reply:
Yes, it's basically trying to avoid running `scrollIntoView` after the component mounts for the first time and only after a field was added, so I think resetting might not be needed. While I say that I realize that `displayedFields` might not be needed as `useEffect` dependency 🤔

Reply:
I'll answer in a comment below.